homeassistant/components/ld2410_ble/binary_sensor.py

```diff
@@ -0,0 +1,71 @@
+"""LD2410 BLE integration light platform."""
+
+
+from homeassistant.components.binary_sensor import (
+    BinarySensorDeviceClass,
+    BinarySensorEntity,
+    BinarySensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers import device_registry as dr
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+)
+
+from . import LD2410BLE
+from .const import DOMAIN
+from .models import LD2410BLEData
+
+ENTITY_DESCRIPTIONS = {
```

Comment:
https://developers.home-assistant.io/docs/core/entity?_highlight=has_entit#entity-naming

has_entity_name True (Mandatory for new integrations)

Reply:
Added.